### PR TITLE
- Use fork height from network properties

### PR DESCRIPTION
**Pull request by rg911**

### CODE REVIEW

File: rest/src/routes/finalizationRoutes.js [hunk 1]
```diff
@@ -35,18 +38,31 @@ module.exports = {
 		const { connections } = services;
 		const { timeout } = services.config.apiNode;
 
+		const readAndParseNetworkPropertiesFile = () => {
+			const readFile = util.promisify(fs.readFile);
+			try {
```
Comment (fboucquez) [inline rest/src/routes/finalizationRoutes.js:43]: Is it possible to move the read of configuration files to a service? 

1) be reused here and in the network routes
2) having the object cached in memory, for some reason the file is being read every request.

File: rest/src/routes/finalizationRoutes.js [hunk 2]
```diff
@@ -35,18 +38,31 @@ module.exports = {
...
+				return readFile(services.config.network.propertiesFilePath, 'utf8')
+					.then(fileData => ini.parse(fileData));
+			} catch (err) {
+				// If API node is not updated, then use static fork height
+				return new Promise(resolve => resolve({ fork_heights: { votingKeyLinkV2: '215500' } }));
```
Comment (fboucquez) [inline rest/src/routes/finalizationRoutes.js:48]: about the hardcoded value, is this for testnet node not updated? In the future, votingKeyLinkV2 will not be provided for testnet/publicnet with the "main" server

**Review by fboucquez — APPROVED** [review]

File: rest/src/routes/finalizationRoutes.js (hunk 2 above)
Comment (rg911) [reply rest/src/routes/finalizationRoutes.js:48]: yeah, this is only for those synced with outdated server code.

File: rest/src/routes/finalizationRoutes.js (hunk 1 above)
Comment (rg911) [reply rest/src/routes/finalizationRoutes.js:43]: thought about it, but this is a simple temporal patch for testnet/v3 only. won't probably need it once mainnet launched.

Comment (fboucquez) [reply rest/src/routes/finalizationRoutes.js:43]: Ok, add a comment "to be removed after main merge" or similar.

**Pull request closed without merging**